dev/scripts/src/alluxio.org/build/cmd/version.go

```diff
@@ -32,17 +33,20 @@ func VersionF() error {
 	return nil
 }
 
-var versionRe = regexp.MustCompile(".*<version>(.*)</version>.*")
+// only one <version> tag is expected in the root pom.xml that is at the level directly under the root project node
+var versionRe = regexp.MustCompile(`^  <version>(.*)</version>$`)
```

Comment:
So this requires the line starts with exactly two spaces?

Reply:
yup exactly two spaces and there's only one of them in the file